opendebates/templates/opendebates/list_ideas.html

```diff
@@ -79,6 +79,7 @@
 {% endblock %}
 
 {% block primary_content %}
+  {% cache 30 idea_list search_term category.id sort %}
```

Comment:
This one might also need `request.GET.page` in the key?  Or something like `{% show_current_number as page_number %}{% cache 30 idea_list search_term category.id sort page_number %}` though I'm not quite sure what the point of that indirection might be. https://django-endless-pagination.readthedocs.org/en/latest/current_page_number.html#in-the-template